assets/queries/ansible/aws/cloudtrail_logging_disabled/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "cloudtrail_logging_disabled",
+  "queryName": "CloudTrail Logging Disabled",
+  "severity": "HIGH",
+  "category": "Logging",
+  "descriptionText": "Enables Logging for the trail",
```

Comment:
The query description should point out the vulnerability being address or how to fix it